MISTAPI.md

```diff
@@ -159,23 +159,23 @@ Sets the main badge of your dapp, right below your dapps menu button.
 
 #### Parameters
 
-1. `String` the string used as the badge text
+1. `String` the string used as the badge text.
 
 ***
 
 ### mist.menu.update(id, [, options] [, callback])
 
-Works like `mist.menu.add()`, but all but the `id` parameters are optional.
+Works like `mist.menu.add()`, but all of the `id` parameters are optional.
```

Comment:
You've changed the meaning of this sentence. All **but** the ID param is optional. Agree that the sentence is clunky, though. Feel free to give it another whirl or revert this change.